packages/create-catalyst/src/utils/clone-catalyst.ts

```diff
@@ -123,7 +132,24 @@ export const cloneCatalyst = async ({
 
   tsConfigJson.compilerOptions.paths['@bigcommerce/components/*'] = ['./components/ui/*'];
 
+  if (!includeFunctionalTests) {
+    tsConfigJson.include = tsConfigJson.include?.filter(
+      (include) => !['tests/**/*', 'playwright.config.ts'].includes(include),
```

Comment:
Waiting to chat as a team to figure out where the tests are going to live. `tests/**/*` is subject to change.